app/src/util/Wallets2Tokens.js

```diff
@@ -15,12 +15,18 @@ class Wallets2Tokens {
     store.state.wallets.portfolioTotal = 0
     // console.log(store.state.currentwallet.config.keys, 'store.state.currentwallet.config.keys ')
     this.tableData = [ ...store.state.currentwallet.config.keys ]
-    /* this.tableData.push({
-      type: 'eos',
+    this.tableData.push({
+      type: 'verto',
       chain: 'eos',
-      name: 'berthonythe2',
+      name: 'EOS HD',
       key: 'c44f57a8dcec3d398b4234dd699891d4db9d7864c14cf1a29e021124f167581f'
```

Comment:
What is this for again?

Reply:
Oops... This was supposed to be commented out.

Test code to replicate the existence of a EOS wallet that isn't associated with any account name yet..